docs/docs-content/user-management/saml-sso/palette-sso-with-okta.md

```diff
@@ -325,6 +326,102 @@ You have now successfully configured Palette SSO based on OIDC with Okta.
 
 
 
+## Okta with SAML
+
+### Create the Okta Application
+
+1. Log in to your Okta Admin console and navigate to **Applications** --> **Applications**. Click the **Create App Integration** button.
+
+  <br />
+
+  :::info
+
+   Your Okta login URL has the following format,
+    `https://{your-okta-account-id}-admin.okta.com/admin/getting-started`. 
+    Replace `{your-okta-account-id}` with your Okta account ID.
+
+  :::
+
+  <br/>
+
+2. In the screen that opens, select **SAML 2.0** for the sign-in method. Then click **Next**.
+
+  <br />
+
+  ![Configure General Settings](/saml-okta-images/okta-create-application.png)
+
+  <br />
+
+1. The following screen allows you to configure the new App Integration. On the **App name** field, change the name from `My Web App` to `Spectro Cloud Palette SAML`. If desired, you can also upload a logo for the application. 
+
+  <br />
+
+  ![Configure General Settings](/saml-okta-images/okta-saml-general-settings.png)
+
+  <br />
+
+4. Open a web browser and navigate to your Palette subscription. Navigate to **Tenant Settings** --> **SSO** and click **SAML**. Click the button next to **Login URL** to copy the value to the clipboard.
+
+5. Set the value of **Service** to **Okta**.
+
+  <br />
+
+  ![Configure General Settings](/saml-okta-images/palette-manage-sso-okta-saml.png)
+
+  <br />
+
+6. Switch back to your Okta Admin console and paste the copied value to the **Single sign-on URL** and **Audience URI (SP Entity ID)**.
+
+7. Specify values within **Attribute Statements** and **Group Attribute Statements** to link user values from Okta to SpectroCloud.
+
+Under **Attribute Statements (Optional)** specify the below values.
+
+| Name | Name Format | Value |
+| -- | -- | -- |
+| `FirstName` | `Unspecified` | `user.firstName` |
+| `LastName` | `Unspecified` | `user.lastName` |
+| `Email` | `Unspecified` | `user.email` |
+
+Under **Group Attribute Statements (Optional)** specify the below values.
+
+| Name | Name Format | Filter | Value |
+| -- | -- | -- | -- |
+| `SpectroTeam` | `Unspecified` | `Matches Regex` | Blank |
+
+  <br />
+
+  ![Configure General Settings](/saml-okta-images/okta-saml-attribute-statements.png)
+
+  <br />
+
+8. Finish the creation of the application with default values.
+
+9. Once brought to main application page, copy the **Metadata URL**, open it up in a separate page, then copy of the contents of the XML.
+
+  <br />
+
+  ![Configure General Settings](/saml-okta-images/okta-metadata-url.png)
+
+  <br />
+
+  :::info
+  Due to some browsers that add additional formatting and spacing for XML data, you can run the below command to copy the contents to your clipboard.
```

Comment:
```suggestion
  Due to some browsers that add additional formatting and spacing for XML data, you can issue the command below to copy the contents to your clipboard.
```

Reply:
@wcrum What if the user does not have `pbcopy` installed? Of if they are on a windows. It might be best to remove that and inform the user to copy the content.  Otherwise, they could get confused why they get an error.